### DISCUSSION TITLE
Recording with depth cameras/binocular stereo camera setup

### DISCUSSION THREAD

**Discussion by theparkerlab** [bonsai-rx/bonsai#2038] [General]
Hello,

I'm interested in setting up video recordings using depth cameras with Bonsai. I know there are two existing Bonsai packages ([RealSense](https://github.com/bonsai-rx/realsense), [Kinect](https://github.com/bonsai-rx/kinect)), however they do not seem to have been modified in the last 8 years. Kinect cameras have been discontinued but can be found used. Newer models of RealSense cameras have been released, and I'm not confident that the current software will work. Both options seem somewhat risky. Has anyone had success with either of these packages, or successfully implemented a stereo depth setup using two regular cameras and OpenCV stereo depth mapping?

Thanks,
Phil

**Comment by banchan86** [comment]
Hi @theparkerlab recently came across the question while answering another one https://github.com/orgs/bonsai-rx/discussions/2087, there is a prerelease version of the RealSense package made in 2021, might want to follow along with that thread to see if it works.
